### ISSUE TITLE
Job Decision

### ISSUE THREAD

**Issue by starkc1** [specfm/design-details#6]
Hi, i am in my fifth and final year of Software Engineering and i have done multiple web development and app development Co-Op/Internships, **i was wondering what the best way or most efficient way to look for and find job opportunities?** I do a lot of Angular, Flutter and Firebase development on my own time, so i like that kind of stuff and i have started in the past year to get into designing the app/website in Figma.
My name is Cameron Stark

**Comment by dcarmona05** [comment]
(not sure if public should answer) - college/school friends working professionally, Linkedin for putting everything together and finding inspiration from outside peers, start attending open houses or just get in contract with sr. recruiters prepared for action (examples of work / who you are).

**Comment by brianlovin** [comment]
@dcarmona05 thanks for the comment! I think it's great if anyone else jumps in and provides perspectives here; @marshallbock and I can include them in the conversation

**Comment by brianlovin** [comment]
Hey Cameron, we discussed this question in today's episode! https://spec.fm/podcasts/design-details/308247

**Comment by starkc1** [comment]
Just listened to it, thank you for taking the time to answer my question. I like hearing another perspective on it and i have been thinking about getting more active in the community with user group meetings as a means to network with people that are potential in the industry.

**Comment by dozornaya** [comment]
Hi Brian & Marshall, 

Just listened to the episode and wanted to follow up on it a little bit. 

Although knowing someone is a great way to get the job, I don't think it's serving anyone any favors by saying that applying on the company website is not an option. Not everyone has a luxury of a network, but everyone needs a job =) There are also different life circumstances, so I consider a skill of cold applying a pretty valuable one. 

I've had 4 career jobs since I moved to US in 2013, and I found 3 of them by applying on Indeed, LinkedIn, or company website. Including switching from design production to UX design last year, and moving from Bay Area to Denver this year. The process of applying online is definitely not easy and can be very disappointing at times. But consider this, even if you have "someone you know", you'll still have to pass the interview and present your portfolio. Just like anything else, talking about yourself, your skills, etc - takes practice. If anything, consider these cold apply instances as a practice. 

And, to your point on the podcast, it might be more efficient, at least sometimes. The first job I got through an online application took 3 mo from the time of application to the first day at work. The second time was harder because I was switching gears, it took about 6 mo to figure out how to tell my story. This last time it only took me a month to find a job in the new city. Technically, I actually got two offers this time around, but the second company got acquired and the offer fell through. 

During the last 2 rounds, I was applying to A LOT of different positions considering most of them as a practice and almost an opportunity to test my resume/portfolio. At the very least it gave me confidence to speak to my skills in those interviews that were actually important to me. 

Maybe not a path for everyone, but an important skill to have. 

Thank you for the great podcast! 
Tamara =)
